app/models/miq_report_result.rb

```diff
@@ -167,24 +167,13 @@ def purge_for_user
   end
 
   def to_pdf
-    page_size = "a4"
-    if report.rpt_options && report.rpt_options[:pdf]
-      page_size = report.rpt_options[:pdf][:page_size] || "a4"
-    end
-
-    curr_tz = Time.zone # Save current time zone setting
-    user = userid.include?("|") ? nil : User.find_by_userid(userid)
-    Time.zone = user ? user.get_timezone : MiqServer.my_server.server_timezone
```

Comment:
@gtanzillo I want to assert that it wasn't (and couldn't see why it was). I'm happy to back out that change for this PR if it's easier to review as this stuff is probably under-tested and timezones tend to make my brain melt a bit